.github/workflows/build-wheels.yml

```diff
@@ -47,4 +47,6 @@ jobs:
 
       - name: Upload wheels to GH
         uses: actions/upload-artifact@v4
-        path: wheelhouse/*.whl
+        with:
+          name: cibw-wheels-${{ matrix.os }}
```

Comment:
I'll let @henryiii comment on different naming choices as there are some that are different (https://github.com/scikit-hep/boost-histogram/blob/develop/.github/workflows/wheels.yml) but LGTM

Reply:
Ah you're using the same format as @henryiii showed in the Scientific Python Developers Guide (https://learn.scientific-python.org/development/guides/gha-wheels/#the-core-job-3-main-oss)

```yaml
    - name: Upload wheels
      uses: actions/upload-artifact@v4
      with:
        name: cibw-wheels-${{ matrix.os }}
        path: wheelhouse/*.whl
```

so that will be fine. Good to go!